src/scss/_main.scss

```diff
@@ -1418,8 +1418,9 @@ nav {
   @include visually-hidden();
 }
 
-.break-all {
+.log-line-word-break {
```

Comment:
i would propose using something more generic like `.break-text` maybe.

Reply:
sounds good, I was just leaning away from the generic stylizers based on previous conversation but Im definitely cool with it 👍

Reply:
done 🚀

Reply:
Ah, I don't recall that. I do remember we talked about eventually creating a utilities import that would have things like this in it. I can see this being useful elsewhere potentially. Thanks!

Reply:
agreed 👍  I'll jot that down in my notes to create a util css file to put all these random classes